python/sglang/srt/server_args.py

```diff
@@ -836,7 +836,7 @@ def add_cli_args(parser: argparse.ArgumentParser):
         parser.add_argument(
             "--enable-flashinfer-mla",
```

Comment:
You can use https://github.com/Fridge003/sglang/blob/06d0538cf1f9ba691c82c37bf55bb9221b133f7b/python/sglang/srt/server_args.py#L1288 for this argument